src/components/App/App.js

```diff
@@ -51,6 +60,8 @@ export default function App() {
         const note = e.target.attributes.note.value;
         synth.triggerAttackRelease(note, '8n');
         toggleActive(note);
+        setCurrentNote(note);
+        setTimeout(() => setCurrentNote(''), 1000);
```

Comment:
I wonder if theres another way to dismount the note, not that this doesn't work perfectly!